### PR TITLE
DC messages

### PR DESCRIPTION
**Pull request by streamer45**
#### Summary

Not much of a change on this side. I removed the `pako` dependency since we now use `fflate`. Also reworked logging and updated dependencies.

#### Related PRs

https://github.com/mattermost/rtcd/pull/158
https://github.com/mattermost/calls-common/pull/40

### CODE REVIEW

File: lt/client/client.go [hunk 1]
```diff
@@ -468,6 +468,10 @@ func (u *User) Speak(text string) chan struct{} {
 }
 
 func (u *User) onConnect() {
+	// Waiting a second before transmitting to avoid
+	// the known race between incoming and outgoing track.
+	// This is needed as pion doesn't support perfect negotiation yet.
+	time.Sleep(time.Second)
```
Comment (streamer45) [inline lt/client/client.go:474]: Using DC further exacerbated the impact of signaling conflicts, so I had to add this to make it usable. Browsers are not affected as they support perfect negotiation (SDP rollback).

Comment (cpoile) [reply lt/client/client.go:474]: Gotcha. :+1:

File: webapp/src/client.ts [hunk 2]
```diff
@@ -5,8 +5,7 @@ import type {EmojiData, CallsClientJoinData} from '@mattermost/calls-common/lib/
 
 import {EventEmitter} from 'events';
 
-// @ts-ignore
-import {deflate} from 'pako/lib/deflate';
+import {zlibSync, strToU8} from 'fflate';
```
Comment (cpoile) [inline webapp/src/client.ts:8]: My own interest, why this change?

**Review by cpoile — APPROVED** [review]

File: webapp/src/client.ts (hunk 2 above)
Comment (streamer45) [reply webapp/src/client.ts:8]: @cpoile It's ~40% faster, proper typescript and 5 times smaller bundle size. It got me convinced.
